index.js

```diff
@@ -92,11 +95,51 @@ getStdin().then(str => {
     ? policy.ports.range
     : null;
 
+  var maxSize = (policy.size)
+    ? policy.size.max
+    : null;
+
+  var warningSize = (policy.size)
+    ? policy.size.warning
+    : null;
+
   var failedEnvKeys = [];
   var failedLabels = [];
   var failedVolumes = false;
   var failedPortRequirement = false;
   var failedPortRange = false;
+  var failedMaxSize = false;
+  var failedWarningSize = false;
+
+  if (warningSize !== null && containerSize >= warningSize) {
```

Comment:
lmao omg `if (bool condition is true) { boo_var = true; }`
You're making [this dude](https://www.binghamton.edu/cs/people/lander.html) cry.